### DISCUSSION TITLE
Saving raw data to a persistent storage on k8s

### DISCUSSION THREAD

**Discussion by pai911** [airbytehq/airbyte#29084] [Connector Questions]
Hi! I read on the description of Local JSON connector that it doesn't support k8s.

I figured it's because raw data gets saved to local storage, and then it goes away with the destination pod.

But since k8s supports PV and PVC to keep storage persistent across pods, there should be a way to use a shared volume to save raw data?

Does Airbyte plan to support saving files directly on a persistent storage?

Any news or sharing is much appreciated!

**Comment by marcosmarxm** [comment]
This is not in the roadmap and I don't think it will prioritize in the future. Local JSON is intended to be an example connector for the Docker deployment. You can always suggest a tech spec implementing this feature.

[a comment by an automated account is not shown]
